main/course_info/about.php

```diff
@@ -165,6 +172,7 @@
 $essence = Essence\Essence::instance();
 $template->assign('essence', $essence);
 $template->assign('is_premium', $courseIsPremium);
+$template->assign('allowSubscribe', $allowSubscribe);
```

Comment:
Should be 'allow_subscribe'

then in the template you can call it {% if allow_subscribe == true %}